heightmap_generator/three.js

```diff
@@ -38,7 +38,7 @@ self.console = self.console || {
 
 		self.requestAnimationFrame = function ( callback ) {
 
-			var currTime = Date.now(), timeToCall = Math.max( 0, 16 - ( currTime - lastTime ) );
+			var currTime = this.time, timeToCall = Math.max( 0, 16 - ( currTime - lastTime ) );
```

Comment:
What is this line doing?  Doesn't seem obvious to me...  Time to call is going to be something between 0 and 16... is that seconds?  Milliseconds? ... after the last time that it was called?  Maybe clearer to do something like:

var nextCallTime = Math.max(currTime, lastTime + 16);

It would also be good to know what the difference between using Date.now() and this.time here.  Seems like it *should* be Date.now, since it's scheduling an event relative to simulation time, not world time.  If we use this.time, do we run the risk of this happening at different times on different systems?  Is that a problem?